### ISSUE TITLE
geom_violin(draw_quantiles) fails with identical y-values

### ISSUE THREAD

**Issue by tklebel** [hadley/ggplot2#1641]
Reprex:

```R
mydata <- data.frame(y = rep(1, 10))

ggplot(mydata, aes(x = 1, y)) +
  geom_violin(draw_quantiles = .5)
# Error in stats::approxfun(data$y, data$xminv) : 
# need at least two non-NA values to interpolate
```

`geom_violin` without `draw_quantiles` and `geom_boxplot` behave correctly in this situation:

``` r
ggplot(mydata, aes(x = 1, y)) +
  geom_violin()
```

![](http://i.imgur.com/Ti5Re0J.png)

**Comment by hadley** [comment]
@DanRuderman can you please take a look?

**Comment by DanRuderman** [comment]
Roger roger. Will review by Monday.

**Comment by DanRuderman** [comment]
Fix pushed and pull requested. Thank you @tklebel for the bug report.

**Comment by thomasp85** [comment]
Fixed with #1687
